### ISSUE TITLE
Add profile to taskbar right click menu

### ISSUE THREAD

**Issue by zapy85** [KurtisLiggett/Simple-IP-Config#51]
Could you add this?
Option to allow us to enable profiles that should be visible when right clicking on the taskbar icon.

**Comment by KurtisLiggett** [comment]
I've thought about doing this before.
What do you think about having 5 'slots' for favorite profiles. You would assign them by right-clicking on a profile.

**Comment by Otiel** [comment]
Good idea. Maybe show as well the current IP config on a tooltip on the systray icon?

**Comment by zapy85** [comment]
Cool, I'm not sure about a 5 row limit though, couldn't it be dynamic or doesn't the au3 format support that?
If not, a 5row limit would be better than nothing :)
Also as Otiel wrote, would be nice to have a static row that shows what profiles is activated on each nic if any is applied (if possible) I tend to name them to what the configs are, ex
ETH0-1.251-GDNS
Has;
Ip, 192.168.1.25
Google DNS, 8.8.8.8

ETH0-DHCP-GDNS
And so on.

**Comment by KurtisLiggett** [comment]
I see your points. My only concern is some users have many profiles, so the menu would end up looking like this..

![image](https://user-images.githubusercontent.com/29669372/32794638-03cd6fce-c92f-11e7-842e-287c99beb561.png)

**Comment by KurtisLiggett** [comment]
Also, could one of you please start a new issue for showing the IP on the tooltip and/or tray menu?

**Comment by Otiel** [comment]
Created #56.

**Comment by zapy85** [comment]
With dynamic i meant the amount of visible profiles should be dynamic and not hard coded to a certain amount, you still would have to choose which profiles you would like to see.
Maybe with a checkbox on each profile?
